tests/check_convert_util.c

```diff
@@ -195,12 +193,11 @@ START_TEST(test_convert_parse_tlvs_multiple){
 	memcpy(buff, tlv1, tlv1_len);
 	memcpy(buff + tlv1_len, tlv2, tlv2_len);
 
-	memset(&opts, '\0', sizeof(struct convert_opts));
-	ret = convert_parse_tlvs(buff, buff_len, &opts);
-	ck_assert_msg(ret == 0, "Should parse multiple TLVs");
-	ck_assert_msg(opts.flags & CONVERT_F_CONNECT,
+	opts = convert_parse_tlvs(buff, buff_len);
```

Comment:
maybe clearer to free it

Reply:
Yep good catch, forgot to free it :-) thx